### DISCUSSION TITLE
Silly NA problem with read.csv

### DISCUSSION THREAD

**Discussion by emmats** [atsa-es/fish550-2023#17] [General]
I have my time series data for my project. There are 3 columns of metadata and 3 columns of observations. 115 rows contain a numerical value in the observation columns; the remaining 25 rows had NA in these 3 columns. When I read the file into R, it only reads in 115 rows of data (it ignores the 25 rows with NA across 3 columns). I have been trying to troubleshoot this, but I cannot figure out what is going on! Attached is my data file and below is my read.csv code.
[metagenome_3taxa_abundance.csv](https://github.com/atsa-es/fish550-2023/files/11294919/metagenome_3taxa_abundance.csv)

metag3<-read.csv('metagenome_3taxa_abundance.csv', na.strings="na")

**Comment by mdscheuerell** [comment]
Hi @emmats. When I peep your data file, I only see 115 rows of data. Here's a screen grab:

<img width="477" alt="Screen Shot 2023-04-21 at 4 50 47 AM" src="https://user-images.githubusercontent.com/5048528/233628848-7d0b4638-f7d7-4e69-bba2-d3ad5cf76143.png">

**Comment by emmats** [comment]
It turns out that I was editing a version saved in a different directory. This is what I get for trying to do work while jet lagged. Thanks for prompting me to find my mistake, Mark!
